sockeye/model.py

```diff
@@ -479,17 +485,38 @@ def load_model(model_folder: str,
     else:
         params_fname = os.path.join(model_folder, C.PARAMS_NAME % checkpoint)
 
+    if (dtype == C.DTYPE_INT8 or model_config.dtype == C.DTYPE_INT8) and "intgemm_fully_connected" not in dir(mx.nd.contrib):
+        #We're going to use int8 but it's not compiled into mxnet.
+        #TODO: configurable option for path!
+        mx.library.load(os.path.abspath("libintgemm.so"))
```

Comment:
Can we put another try-except around this statement in case both fails (1. loading an mxnet version with intgemm included, and loading the custom operator .so file)?

Reply:
Fixed in ee9b05d6